tower/src/make/make_service.rs

```diff
@@ -65,3 +147,51 @@ where
         Service::call(self, target)
     }
 }
+
+#[derive(Debug, Clone)]
+pub struct IntoService<M, Request> {
```

Comment:
let's make sure to add doc comments for these structs as well. it's fine if they just say 
```suggestion
/// Service returned by [`MakeService::into_service`][into]. 
/// 
/// See the documentation on [`into_service`][into] for details.
///
/// [into]: MakeService::into_service
#[derive(Debug, Clone)]
pub struct IntoService<M, Request> {
```
or something